### PR TITLE
Add support for staking contract

### LINKED ISSUE #9: Add support for staking contract
### Description

Add support for staking contract

### Acceptance Criteria

- Implement selector and handlers for staking contract functions
- Update readme

### Additional Information
[Staking contract](https://sepolia-blockscout.lisk.com/address/0x456fb1620Fa2Bba50Dc75df675Fddff0082e92D5)

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #9 #10 

### How was it solved?

- [x] Implement selector and handlers for reward contract functions
  - [x] createPosition
  - [x] initiateFastUnlock
  - [x] claimRewards
  - [x] pauseUnlocking
  - [x] resumeUnlockingCountdown
  - [x] increaseLockingAmount
  - [x] extendDuration 
- [x] Update readme
- [x] Add unit tests for all selectors

### How was it tested?

- Local:
  - [x] By generating snapshots and running tests (Ledger VS code extension)

### CODE REVIEW

**Comment by nagdahimanshu** [comment]
CI is failing and its a issue from [oss-fuzz](https://github.com/google/oss-fuzz/issues/11886).

File: .clang-format [hunk 1]
```diff
@@ -7,7 +7,7 @@ ColumnLimit: 100
 PointerAlignment: Right
 AlignAfterOpenBracket: Align
 AlignConsecutiveMacros: true
-AllowAllParametersOfDeclarationOnNextLine: false
+# AllowAllParametersOfDeclarationOnNextLine: false
```
Comment (shuse2) [inline .clang-format]: why this is changed?

File: PLUGIN_SPECIFICATION.md [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 
 Smart contracts covered by this plugin are:
 
-|  Network | Version | Smart Contract | Address |
-|   ----   |   ---   |      ----      |   ---   |
-| Lisk Sepolia Testnet     | -  | TokenClaim  | `0x5c3a68B5C635Ce0DA7648C30A1B83A61C376bd87` |
+| Network              | Version | Smart Contract | Address                                        |
+| -------------------- | ------- | -------------- | ---------------------------------------------- |
+| Lisk Sepolia Testnet | -       | TokenClaim     | `0x4bf6F67f7B1EF8C7a6C9C7a202C8b6838a3490e4`   |
+| Lisk Sepolia Testnet | -       | Reward         | `0x0xfB072D568840BC67846B24d64D66095C1f932F99` |
```
Comment (shuse2) [inline PLUGIN_SPECIFICATION.md]: Official addresses are here https://github.com/LiskHQ/lisk-contracts/tree/main/deployment/addresses

File: .clang-format (hunk 1 above)
Comment (nagdahimanshu) [reply .clang-format]: There is a duplicate entry for `AllowAllParametersOfDeclarationOnNextLine` property in this file. I'll remove it.

**Review by hrmhatef — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**